remote-api/src/main/java/org/alfresco/rest/api/impl/rules/ActionParameterConverter.java

```diff
@@ -89,6 +91,21 @@ Map<String, Serializable> getConvertedParams(Map<String, Serializable> params, S
         return parameters;
     }
 
+    public Serializable convertParamFromServiceModel(Serializable param)
+    {
+        if (param instanceof QName)
+        {
+            return ((QName) param).toPrefixString(namespaceService);
+        }
+        else if (param instanceof NodeRef) {
+            return param.toString();
```

Comment:
Is this just returned as a node id (for V1 style API) ? (in other words, the store ref is assumed to be workspace://SpacesStore for V1 Nodes API).

Reply:
It is returned as `workspace://SpacesStore/node-id` - same way as it is expected as an input param.
Shall I change that to accept/return `nodeId` only?

Reply:
Changed to accept/return `nodeId` only.